rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -595,10 +590,9 @@ impl Parse for DistributionsRawRow {
             tags_key: tag_keys.iter().map(|k| k.parse::<u64>().unwrap()).collect(),
             tags_indexed_value: vec![0; tag_keys.len()],
             tags_raw_value: tag_values,
-            materialization_version: parse_dist_materialization_version(
-                get_str_config("gen_metrics_dist_mat_version"),
-                "distribution".to_string(),
-            ),
+            materialization_version: parse_matview_ver_from_runtime_opt("distribution")
+                .unwrap_or(3),
```

Comment:
IIUC this is setting the default to version 3 for distributions? That makes sense to me

Reply:
Yup, thanks for double checking!

Reply:
nit; I just noticed that the name of the old runtime config was `gen_metrics_dist_mat_version` in case you want to use that. then it would be `dist` instead of `distribution`

Reply:
Yeah that is intentional. I don't want the old set of runtime config to take effect here. This should be an entirely new set of runtime config to prevent confusion. Also I think the old flag is set to 3, which is the default value here